src/boot/efi/console.c

```diff
@@ -149,9 +149,69 @@ static EFI_STATUS change_mode(UINTN mode) {
         return err;
 }
 
+static UINT64 text_area_from_font_size()
+{
```

Comment:
coding style. in C a function with no parameters, is specified with a "void", unnamed parameter. also, please place opening { on the same line as the function name:

```c
static UINT64 text_aread_from_font_size(void) {
…
```

Reply:
ok